synapse/events/spamcheck.py

```diff
@@ -45,3 +45,19 @@ def check_event_for_spam(self, event):
             return False
 
         return self.spam_checker.check_event_for_spam(event)
+
+    def user_may_invite(self, userid):
```

Comment:
I wonder if we also want to include the room_id here, so that you can fonx all invites for known bad rooms (or whitelist known good rooms)

Reply:
mm, could be useful